Sources/Examples/ConformanceMismatches.swift

```diff
@@ -0,0 +1,144 @@
+import Library
+
+// MARK: Under-Specified Protocol
+
+#if swift(<6.0)
+/// A conforming type that has now adopted global isolation.
+@MainActor
+class WindowStyler: Styler {
+	// Swift 5 Warning: main actor-isolated instance method 'applyStyle()' cannot be used to satisfy nonisolated protocol requirement
+	// Swift 6 Error: main actor-isolated instance method 'applyStyle()' cannot be used to satisfy nonisolated protocol requirement
+	func applyStyle() {
+	}
+}
+#endif
+
+// MARK: Globally-Isolated Protocol
+
+/// A conforming type that will infer the protocol's global isolation.
+class GloballyIsolatedWindowStyler: GloballyIsolatedStyler {
+	func applyStyle() {
+	}
+}
+
+/// A conforming type that will infer the protocol's global isolation *only*
+/// for the satisfying methods.
+class PerRequirementIsolatedWindowStyler: PerRequirementIsolatedStyler {
+	func applyStyle() {
+		// only this is MainActor-isolated
+	}
```

Comment:
And then add a method here that is not inferred main actor:

```suggestion
	}
		
		func synchronousMethod() {
 		// this method is non-isolated; it is not witnessing any isolated protocol requirement
 	}
```

Reply:
yeah this is a particularly useful thing to highlight